dbt_subprojects/hourly_spellbook/macros/sector/bridges/across_v3_withdrawals.sql

```diff
@@ -0,0 +1,50 @@
+{% macro across_v3_withdrawals(blockchain, events) %}
+
+WITH across_id_mapping AS (
+    SELECT id, blockchain
+    FROM (VALUES
+    (1, 'ethereum')
+    , (10, 'optimism')
+    , (137, 'polygon')
+    , (42161, 'arbitrum')
+    , (56, 'bnb')
+    , (324, 'zksync')
+    , (59144, 'linea')
+    , (8453, 'base')
+    , (7777777, 'zora')
+    , (81457, 'blast')
+    , (34443, 'mode')
+    , (232, 'lens')
+    , (57073, 'ink')
+    , (1135, 'list')
+    , (41455, 'aleph_zero')
+    , (690, 'redstone')
+    , (534352, 'scroll')
+    , (1868, 'soneium')
+    , (480, 'worldchain')
+    , (130, 'unichain')
+    ) AS x (id, blockchain)
+    )
```

Comment:
probably better to add this in one `view` model to not replicate static list. (not blocking)